content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -130,119 +103,97 @@ spec:
-
-### GKE setup
-
-1.  Refer to the procedure to [prepare a GKE cluster for Istio](/docs/setup/platform-setup/gke/) and
-    enable [network-policy](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy) in your cluster.
-
-    {{< warning >}}
-    For existing clusters, this redeploys all nodes.
-    {{< /warning >}}
+* Red Hat OpenShift 4.2+
 
-1.  Install Istio CNI via `Istioctl` including the `--set values.cni.cniBinDir=/home/kubernetes/bin` option.
-    For example, the following `istioctl manifest` command sets the `values.cni.cniBinDir` value for a GKE cluster:
-
-    {{< text bash >}}
-    $ istioctl install --set values.cni.cniBinDir=/home/kubernetes/bin \
-        --set components.cni.enabled=true \
-        --set components.cni.namespace=kube-system
-    {{< /text >}}
-
-## Sidecar injection compatibility
-
-The use of the Istio CNI plugin requires Kubernetes pods to be deployed with a sidecar injection method
-that uses the `istio-sidecar-injector` configmap created from the installation with the
-`--set cni.enabled=true` option.  Refer to [Istio sidecar injection](/docs/setup/additional-setup/sidecar-injection/)
-for details about Istio sidecar injection methods.
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    cni:
+      enabled: true
+      namespace: kube-system
+  values:
+    sidecarInjectorWebhook:
+      injectedAnnotations:
+        k8s.v1.cni.cncf.io/networks: istio-cni
+    cni:
+      cniBinDir: /var/lib/cni/bin
+      cniConfDir: /etc/cni/multus/net.d
+      cniConfFileName: istio-cni.conf
+      chained: false
+{{< /text >}}
 
-The following sidecar injection methods are supported for use with the Istio CNI plugin:
+## Operation details
 
-1.  [Automatic sidecar injection](/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection)
-1.  Manual sidecar injection with the `istio-sidecar-injector` configmap
-    1.  [`istioctl kube-inject`](/docs/reference/commands/istioctl/#istioctl-kube-inject) using the configmap directly:
+### Upgrade
 
-        {{< text bash >}}
-        $ istioctl kube-inject -f deployment.yaml -o deployment-injected.yaml --injectConfigMapName istio-sidecar-injector
-        $ kubectl apply -f deployment-injected.yaml
-        {{< /text >}}
+When upgrading Istio with [in-place upgrade](/docs/setup/upgrade/in-place/),
+CNI component can be upgraded together with the control plane using one `IstioOperator` resource.
 
-    1.  `istioctl kube-inject` using a file created from the configmap:
+When upgrading Istio with [canary upgrade](/docs/setup/upgrade/canary/), since CNI component runs as a cluster singleton,
+it is recommended to operate and upgrade CNI separately from the revisioned control plane.
+The following `IstioOperator` can be used to operate CNI component independently.
```

Comment:
```suggestion
When upgrading Istio with [canary upgrade](/docs/setup/upgrade/canary/), because the CNI component runs as a cluster singleton,
it is recommended to operate and upgrade the CNI component separately from the revisioned control plane.
The following `IstioOperator` can be used to operate the CNI component independently.
```